js/lib/DateRngFmt.js

```diff
@@ -372,6 +374,10 @@ DateRngFmt.prototype = {
         monthTemplate = this.dateFmt._tokenize(this.dateFmt._getFormatInternal(formats, "m", this.length) || "MM");
         dayTemplate = this.dateFmt._tokenize(this.dateFmt._getFormatInternal(formats, "d", this.length) || "dd");
 
+        if (isStandAlone && monthTemplate[0].length >= 3) {
```

Comment:
Need a comment here to explain why this is. I can just imagine that a few years from now, we are looking at this and thinking to ourselves, "What is this?!?!"

Reply:
yes, you're right. I added an explanation and updated version number.